src/Components/WebView/WebView/src/IpcReceiver.cs

```diff
@@ -46,8 +46,6 @@ public async Task OnMessageReceivedAsync(PageContext pageContext, string message
                 // For any other message, you have to have a page attached already
                 if (pageContext == null)
                 {
-                    // TODO: Should we just ignore these messages? Is there any way their delivery
-                    // might be delayed until after a page has detached?
```

Comment:
I don't think there's any concern here. Nothing would go wrong unless we receives one of these messages before the first page connection, and that's impossible, because it's the page that sends them.

Reply:
Hopefully the exception here will keep us honest!